macros/stitch/base/stitch_fb_ad_creatives.sql

```diff
@@ -8,39 +8,55 @@
 {% macro default__stitch_fb_ad_creatives() %}
```

Comment:
almost all of the implementation of this model across redshift and snowflake is identical. we should refactor this to make the code more DRY.